Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -886,6 +1081,48 @@
     return isFullFramePlugin();
 }
 
+// Accessibility Related
+bool PDFPluginBase::showContextMenuAtPoint(const IntPoint& point)
+{
+    auto* frameView = m_frame ? m_frame->coreLocalFrame()->view() : nullptr;
+    if (!frameView)
+        return false;
+    IntPoint contentsPoint = frameView->contentsToRootView(point);
+    WebMouseEvent event({ WebEventType::MouseDown, OptionSet<WebEventModifier> { }, WallTime::now() }, WebMouseEventButton::Right, 0, contentsPoint, contentsPoint, 0, 0, 0, 1, WebCore::ForceAtClick);
+    return handleContextMenuEvent(event);
+}
+
+IntPoint PDFPluginBase::convertFromPDFViewToRootView(const IntPoint& point) const
```

Comment:
I feel like we decided a while back that "PDFView" coordinate space didn't make sense in the generic case (that it's a PDFPlugin-specific thing). Might this match one of our existing coordinate space conversion methods? If not, I think we need a different name.

Reply:
Current coordinate space is a little off, but I kinda want to solve all the coordinate problems all at once. I think I would push a pr later to solve this problem.

Reply:
Please add a FIXME for this here.